kbase-extension/scss/partials/_narrative.scss

```diff
@@ -980,15 +985,7 @@ p#site-title {
   font-weight: 700;
   text-align: center;
   user-select: none;
-}
 
-.kb-overlay-active {
```

Comment:
this is in the middle of two `.kb-side-header` stanzas, so I moved it out and combined the two